### PR TITLE
v1.15 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Highlights
+ - Added mention and reply mention highlighting.
+ - Added a sliding animation for the reaction list.
+ - Added support for pinned posts.
+ - Added support for jumboemojis.
```
Comment (saturninoabril) [inline CHANGELOG.md]:
```suggestion
 - Added support for jumbo emojis.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+### Highlights
+ - Added mention and reply mention highlighting.
+ - Added a sliding animation for the reaction list.
+ - Added support for pinned posts.
+ - Added support for jumbo emojis.
+ - Added support for interactive dialogs.
+ - Improved UI for the long press menu and emoji reaction viewer
+
+### Improvements
+ - Added the ability to include custom headers with requests.
```
Comment (enahum) [inline CHANGELOG.md]: this is with custom builds

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+ - Added support for clearing notifications from a specific channel.
```
Comment (enahum) [inline CHANGELOG.md]: I wouldn't say support, this should be a statement: Push Notifications are grouped by channels are cleared once the channel is read, or something like that

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+ - Added support for auto-reconnecting when unable to reach the server.
```
Comment (enahum) [inline CHANGELOG.md]:
```suggestion
- Improved auto-reconnect when unable to reach the server.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+ - Added support for changing the mobile client status to offline when the app loses connection.
+ - Added 'View Members' button to archived channels.
+ - Added support for keeping the postlist in place without scrolling when new content is available.
```
Comment (enahum) [inline CHANGELOG.md]: this is iOS only

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+
+### Bug Fixes
+ - Fixed an issue where clicking on a file did not show downloading progress.
+ - Fixed an issue where deleting a post caused an app crash.
```
Comment (enahum) [inline CHANGELOG.md]: the previous release did not have this issue, so we can remove this one

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,43 @@
...
+ - Fixed an issue on Android where on fresh install the share extension would not properly show available channels.
+ - Fixed an issue where recently archived channels remained in in: autocomplete when they had been archived.
+ - Fixed an issue where text should render when no actual custom emoji matched the named emoji pattern.
+ - Fixed an issue where attempting to create a channel with an existing name blanked the modal.
```
Comment (enahum) [inline CHANGELOG.md]: same with this one

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,43 @@
...
+ - Fixed an issue on iOS where text got cut-off after replying to a message.
+ - Fixed an issue on Android where the share extension did not properly show available channels on a fresh install.
```
Comment (enahum) [inline CHANGELOG.md]: duplicate

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,43 @@
...
+ - Fixed an issue where search modifier for channels was showing Direct Messages without usernames.
+ - Fixed an issue where "Close Channel" did not work properly when viewing two archived channels in a row.
+ - Fixed an issue with "Critical Error" screen when trying to upload a file from "+" to the left of message input box.
```
Comment (enahum) [inline CHANGELOG.md]: certain file types

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,43 @@
...
+ - Fixed an issue on iOS where text was being cutoff after replying to a message.
```
Comment (enahum) [inline CHANGELOG.md]: duplicate

**Comment by amyblais** [comment]
@enahum / @saturninoabril Can one of you help merge this PR today?

File: CHANGELOG.md (hunk 4 above)
Comment (enahum) [reply CHANGELOG.md]: @amyblais this should be address before merging

**Comment by amyblais** [comment]
@enahum / @saturninoabril Reminder to merge this PR today

**Pull request merged**